apps/site/layouts/layouts.module.css

```
@@ -228,9 +228,8 @@
 
     main {
       @apply max-w-[660px]
-        gap-4
-        break-all
-        lg:break-normal;
+        gap-4;
+      overflow-wrap: anywhere;
```

small indentation fix

```suggestion
        gap-4;
        overflow-wrap: anywhere;
```